components/Search.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+import React from "react";
+import SearchBar from "./SearchBar";
+import SearchResult from "./SearchResult";
+
+export default function Search() {
+  const [searchValue, setSearchValue] = React.useState<string>("");
+
+  return (
+    <div>
```

Comment:
you can just use a document fragment here

```suggestion
    <>
```